activities.json

```diff
@@ -506,6 +506,18 @@
     "title": "HTTP Client Hints",
     "url": "https://tools.ietf.org/html/draft-ietf-httpbis-client-hints"
   },
+  {
+    "ciuName": null,
+    "description": "The ES Import Attributes and JSON modules proposal adds an inline syntax for module import statements to pass on more information alongside the module specifier, and an initial application for such attributes in supporting JSON modules in a common way across JavaScript environments.",
+    "id": "import-attributes",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1648614",
+    "mozPosition": "worth prototyping",
+    "mozPositionDetail": "This proposal enables importing non-js files into a JS module, specifically in this case JSON. Without this mechanism, it wouldn't be possible to do this in a way that isn't open to midflight changing of the file type, which could then be inteereepreetteed as JS. The proposal is necessary to move forward on HTML/CSS/JSON modules. We consider it important that this is defined inbound rather than out of bound.",
```

Comment:
```suggestion
    "mozPositionDetail": "This proposal enables importing non-js files into a JS module, specifically in this case JSON. Without this mechanism, it wouldn't be possible to do this in a way that isn't open to midflight changing of the file type, which could then be interpreted as JS. The proposal is necessary to move forward on HTML/CSS/JSON modules. We consider it important that this is defined inbound rather than out of bound.",
```